src/vs/workbench/parts/terminal/electron-browser/terminalActions.ts

```diff
@@ -380,8 +385,10 @@ export class SplitInActiveWorkspaceTerminalAction extends Action {
 		if (!instance) {
 			return Promise.resolve(void 0);
 		}
-		this._terminalService.splitInstance(instance);
-		return this._terminalService.showPanel(true);
+		return instance.getCwd(this._terminalService.configHelper).then(cwd => {
+			this._terminalService.splitInstance(instance, { cwd: cwd });
```

Comment:
You can use the shorthand here of `{ cwd }` because the key and variable are the same